packages/primitives/src/icons/briefcase-16.svg

```diff
@@ -0,0 +1,21 @@
+<!--
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
We should not use the phrase "All Rights Reserved" in open source license headers. Please check the other places as well.